src/Pages/Questions/Questions.jsx

```diff
@@ -55,14 +55,30 @@ function Questions() {
                   : "each_option"
               }`}
               key={eachOption.value}
-              onClick={() => optionHandler(eachOption.value)}
+              onClick={() => optionHandler(eachOption.value, eachOption)}
             >
               {eachOption.value}
             </div>
           ))}
-          <button className="btn" onClick={nextQuesHandler}>
-            Next &nbsp; <i className="angleRight"><BsArrowRightCircleFill/></i>
-          </button>
+          <div>
+            {ques === 4 ? (
+              <Link to="/ResultPage">
+                <button className="viewScorebtn">
```

Comment:
Usually, classes are not named in camel case. Hyphen separated class names are more common.